tests/osgi-tests/src/test/java/org/wso2/carbon/deployment/engine/osgi/utils/OSGiTestUtils.java

```diff
@@ -119,7 +70,6 @@
      * Set the environment prior to tests.
      */
     public static void setEnv() {
-        setCarbonHome();
         setStartupTime();
         copyCarbonYAML();
```

Comment:
With the new pax exam container, if you want to change any configuration files, you have to use copyfile option, refer,
https://github.com/wso2/carbon-kernel/blob/master/tests/osgi-tests/src/test/java/org/wso2/carbon/osgi/runtime/CarbonRuntimeOSGiTest.java#L119